textile/features.textile

```diff
@@ -703,7 +703,7 @@ h3(#realtime-channel). RealtimeChannel
 *** @(RTL6i3)@ Allows @name@ and or @data@ to be @null@.  If any of the values are @null@, then key is not sent to Ably i.e. a payload with a @null@ value for @data@ would be sent as follows @{ "name": "click" }@
 ** @(RTL6c)@ Connection and channel state conditions:
 *** @(RTL6c1)@ If the connection is @CONNECTED@ and the channel is neither @SUSPENDED@ nor @FAILED@ then the messages are published immediately
-*** @(RTL6c2)@ If the connection is @INITIALIZED@, @CONNECTING@ or @DISCONNECTED@; and the channel is neither @SUSPENDED@ nor @FAILED@; and @ClientOptions#queueMessages@ has not been explicitly set to false; then the message will be placed in a connection-wide message queue to be delivered as soon as the connection is @CONNECTED@. (The recommended implementation is to have the message be sent from the channel to the connection if it fulfils the channel state condition; the connection can then dispatch, queue, or reject it according to its own state and @queueMessages@)
+*** @(RTL6c2)@ If the connection is @INITIALIZED@, @CONNECTING@ or @DISCONNECTED@; and the channel is neither @SUSPENDED@ nor @FAILED@; and @ClientOptions#queueMessages@ is @true@; then the message will be placed in a connection-wide message queue to be delivered as soon as the connection is @CONNECTED@. (The recommended implementation is to have the message be sent from the channel to the connection if it fulfils the channel state condition; the connection can then dispatch, queue, or reject it according to its own state and @queueMessages@)
```

Comment:
I guess the statements were phrased in such a way because default value is `true`. 
But anyways, simplification looks good.

Reply:
Cool, gonna resolve this comment